extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeCommentsExtractor.java

```diff
@@ -257,4 +257,17 @@ public boolean isCommentsDisabled() throws ExtractionException {
 
         return optCommentsDisabled.get();
     }
+
+    @Override
+    public int getCommentsCount() throws ExtractionException {
+        final JsonObject commentsHeaderRenderer = ajaxJson
+                .getArray("onResponseReceivedEndpoints").getObject(0)
+                .getObject("reloadContinuationItemsCommand")
+                .getArray("continuationItems").getObject(0)
+                .getObject("commentsHeaderRenderer");
+
+        final String text = getTextFromObject(commentsHeaderRenderer.getObject("countText"));
```

Comment:
I think we don't need a variable here.

* ``commentsHeaderRenderer.getObject("countText")`` can be done in the above code block
* ``getTextFromObject`` can be done inline in the below code